lib/surface/compiler.ex

```diff
@@ -465,9 +465,17 @@ defmodule Surface.Compiler do
          :ok <- validate_properties(mod, attributes, directives, meta) do
       expanded = mod.expand(attributes, children, meta)
 
+      compile_dep_expr = %AST.Expr{
+        value:
+          quote generated: true, line: node_meta.line do
+            require(unquote(mod)).__compile_dep__()
+          end,
+        meta: %AST.Meta{}
```

Comment:
@lnr0626 I'm not sure if I should use `node_meta.line` and `%AST.Meta{}` here but since this call is not supposed to fail, maybe it doesn't matter :)

Reply:
i /think/ we might want to use `meta: node_meta` instead of `meta: %AST.Meta{}`

Reply:
@lnr0626 ended up using the `meta` which was already a `%Surface.AST.Meta{}`. The `node_meta` was just a map with value `%{line: line}` with `line` being the line in the template, not the file.